website/docs/guides/advanced-cluster-new-sharding-schema.html.markdown

```diff
@@ -0,0 +1,344 @@
+---
+page_title: "advanced_cluster - Migration to new sharding schema and leveraging Independent Shard Scaling"
+---
+
+**Objective**: Guide users to migrate their existing advanced_cluster configurations to use the new sharding schema which was introduced in version `1.18.0`. Additionally, a section is included to describe how Independent Shard Scaling can be used once the new sharding schema is adopted. Exiting sharding configurations will continue to work but will have deprecation messages if not using the new sharding schema.
+
+- [Overview of schema changes](#overview)
+- [Migrating existing advanced_cluster type SHARDED](#migration-sharded)
+- [Migrating existing advanced_cluster type GEOSHARDED](#migration-geosharded)
+- [Migrating existing advanced_cluster type REPLICASET](#migration-replicaset)
+- [Leveraging Independent Shard Scaling](#leveraging-iss)
+
+<a id="overview"></a>
+# Overview of schema changes
+
+`replication_specs` attribute has been modified to now be able to represent each individual shard of a cluster with a unique replication spec element. This implies that when using the new sharding schema the existing attribute `num_shards` will no longer be defined, and instead the number of shards will be defined by the number of `replication_specs` elements.
+
+<a id="migration-sharded"></a>
+## Migrating existing advanced_cluster type SHARDED
+
+Considering the following configuration of a SHARDED cluster using the deprecated `num_shards`:
+```
+resource "mongodbatlas_advanced_cluster" "test" {
+  project_id   = var.project_id
+  name         = "SymmetricShardedCluster"
+  cluster_type = "SHARDED"
+
+  replication_specs {
+    # deprecation warning will be encoutered for using num_shards
+    num_shards = 2 
+    region_configs {
+      electable_specs {
+        instance_size = "M30"
+        disk_iops = 3000
+        node_count    = 3
+      }
+      provider_name = "AWS"
+      priority      = 7
+      region_name   = "EU_WEST_1"
+    }
+  }
+}
+```
+
+In order to update our configuration to the new schema we will remove the use of `num_shards` and add a new identical `replication_specs` element for each shard. Note that these 2 changes must be done at the same time.
```

Comment:
missing comma: In order to update our configuration to the new schema, we will ...